services/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = "",
+        first_name: str = "",
+        last_name: str = ""
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(pk=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = "",
+        password: str = "",
+        email: str = "",
+        first_name: str = "",
+        last_name: str = ""

```

Comment:
I think using None would make the code more readable and also would take slightly less memory)

Reply:
some tests fail if i using None

Reply:
fail test_user_service_create_user

Reply:
i'm sorry, i tried using None on 9-11 lines. My bad